Source/WebCore/platform/graphics/ImageSource.h

```diff
@@ -84,28 +85,28 @@ class ImageSource final
     void setFrameDecodingDurationForTesting(Seconds duration) { m_frameDecodingDurationForTesting = duration; }
     Seconds frameDecodingDurationForTesting() const { return m_frameDecodingDurationForTesting; }
 
+    EncodedDataStatus encodedDataStatus() const;
+    WEBCORE_EXPORT IntSize size(ImageOrientation = ImageOrientation::Orientation::FromImage) const;
+    IntSize sourceSize(ImageOrientation = ImageOrientation::Orientation::FromImage) const;
+    bool hasDensityCorrectedSize() const { return m_metadata.densityCorrectedSize().has_value(); }
+    ImageOrientation orientation() const { return m_metadata.orientation(); }
+    WEBCORE_EXPORT unsigned frameCount() const;
+    RepetitionCount repetitionCount() const { return m_metadata.repetitionCount(); }
+    DestinationColorSpace colorSpace() const { return m_metadata.colorSpace(); }
+    bool notSolidColor() const;
```

Comment:
`notSolidColor()` is an odd name for a boolean getter. We usually use a prefix, per coding style. E.g. `isNotSolidColor()`. The double negative is also not great, not sure if this is avoidable. Maybe `hasSolidColor()` and we reverse the bool?